emulator/src/kreator/preprocessor.rs

```diff
@@ -150,14 +153,12 @@ fn replace_macros(code: &Vec<String>) -> Result<Vec<String>, &'static str> {
                     }
                     macroless_code.push(line.trim().to_string());
                 }
+                continue 'outer;
```

Comment:
alternatively, you could use break here, then the label would not be necessary. It's fine either way though so your call